src/application.factory.ts

```diff
@@ -37,4 +46,33 @@ export const createApp = (configureBindings: (container: Container) => void = ()
       addFilters(env)
     })
     .build()
+}
+
+// tslint:disable-next-line: no-empty
```

Comment:
And this one?

Reply:
This function is optional - if it is not passed as an argument, it will do nothing. Typescript has a problem with that.